hazelcast/src/main/java/com/hazelcast/map/impl/operation/GetAllOperation.java

```diff
@@ -72,8 +71,7 @@ protected void evict() {
         if (recordStore == null) {
             return;
         }
-        final long now = Clock.currentTimeMillis();
-        recordStore.evictEntries(now);
+        recordStore.evictEntries();
```

Comment:
There may be map-loader configured, and we may exceed the threshold due to loading of the missing entries.